File: api/Sources/api/Enums/host_commands.swift
enum HostCommands: String {
    case RECEIVABLE_SET = "set"
    case RECEIVABLE_START = "start"
    case RECEIVABLE_END = "end"

    case SENDABLE_START = "host$start"
    case SENDABLE_END = "host$end"
}

/*
Host -> Join
[Host]

notify: Host
-> joined: Host

Player -> Join
[Host, Player]


*/

File: android/app/src/main/java/br/dev/marconi/lyricalia/enums/HostCommands.kt
package br.dev.marconi.lyricalia.enums

class HostCommands {
    companion object {
        const val ENTITY = "host"
        const val RECEIVABLE_END = "end"
        const val RECEIVABLE_START = "start"

        // SENDABLES
        fun SET(userId: String) = "host\$set$${userId}"
        const val START = "host\$start"

        // RECEIVABLES
        const val SENDABLE_END = "host\$end"
    }
}